app/assets/stylesheets/_vocab_sheet.scss

```diff
@@ -297,3 +306,22 @@
 .size_controls .selected_icon {
   background-color: #d2e2f9;
 }
+
+p.print_name {
```

Comment:
Are these all print specific styles? If so they should probably go into the print style sheet

Reply:
yes you are right.  Thank you : )